pkg/services/rendering/rendering.go

```diff
@@ -61,6 +61,12 @@ func (rs *RenderingService) Init() error {
 		return fmt.Errorf("failed to create images directory %q: %w", rs.Cfg.ImagesDir, err)
 	}
 
+	// ensure CSVsDir exists
+	err = os.MkdirAll(rs.Cfg.CSVsDir, 0700)
```

Comment:
Just a question, but have you considered if e.g. the owning group should have access to this directory?

Reply:
I must admit that I took the same permissions than for the images directory. I don't see why these two directories should have different permissions though.

Reply:
Sounds good.